### ISSUE TITLE
Operadores lógicos: respuesta correcta marcada como errónea

### ISSUE THREAD

**Issue by ByDSA** [midudev/aprende-javascript-problemas#361]
Link: https://www.aprendejavascript.dev/clase/introduccion/operadores-logicos

En el ejercicio de "Tenemos un producto en una tienda. Cuesta 1500 y tenemos un descuento del 25%. Tengo 1150€ en mi cartera. Escribe un código que me diga si puedo comprarlo. No uses paréntesis." he puesto como respuesta "1150 - 1500 * 0.75 >= 0", siendo equivalente a la respuesta sugerida ("1500 * 0.75 <= 1150") y me la marca como respuesta incorrecta. Sin embargo sí marca como correcta "1150 >= 1500 * 0.75" (que es correcto).

También, notificar que el botón de "Ver solución" sólo funciona una vez. La siguiente vez el texto no se sustituye por la respuesta correcta pero el notificador de correcta/errónea se pasa a "correcta" independientemente de lo que haya en el textbox.

**Comment by mateorol9** [comment]
@ByDSA Si entiendo bien el ejercicio que planteas está malo, te lo explico: 1500 * 0.75 - 1150 >= 0 da como resultado -25, en otras palabras es incorrecto. Aun así deberías es plantearlo de esta manera para que te de True: 1150 - 1500 * 0.75 >= 0. Pd: ¡Aun así da incorrecto el ejercicio con este planteamiento, saludos!

**Comment by ByDSA** [comment]
> @ByDSA Si entiendo bien el ejercicio que planteas está malo, te lo explico: 1500 * 0.75 - 1150 >= 0 da como resultado -25, en otras palabras es incorrecto. Aun así deberías es plantearlo de esta manera para que te de True: 1150 - 1500 * 0.75 >= 0. Pd: ¡Aun así da incorrecto el ejercicio con este planteamiento, saludos!

Sí, cometí un typo al escribirlo en el issue, ya lo he editado. Gracias ;)

**Comment by xxiicode** [comment]
Es que la respuesta correcta seria **1500 - 1500 * 0.25 <= 1150**.
actualmente te dice que lo correcto es 1500 * 0.25 <= 1150,  y eso seria preguntar si 375 es menor que 1150. 375 es solamente el valor del descuento.
